lib/net/server/libp2pserver.ts

```diff
@@ -99,7 +98,8 @@ export class Libp2pServer extends Server {
         })
       })
     }
-    (this.node as Libp2pNode).on('peer:discovery', async (peerInfo: any) => {
+    // eslint-disable-next-line no-extra-semi
+    ;(this.node as Libp2pNode).on('peer:discovery', async (peerInfo: any) => {
```

Comment:
There were 8 locations where prettier requires we insert a semicolon to prevent [potential ASI failures][1]. However eslint (correctly?) complains these are unnecessary - e.g. their views conflict. 

Deferred to prettier in this case because otherwise its semi rule would need to be turned off.  (Not sure this is the right way thing though...)

[1]: https://prettier.io/docs/en/rationale.html#semicolons